Ginger/GingerCoreNET/ActionsLib/ActFileOperations.cs

```diff
@@ -341,6 +346,11 @@ private void SetupDestinationfolders()
             string calculatedDestinationPath = GetInputParamCalculatedValue(Fields.DestinationFolder);
 
             calculatedDestinationPath = WorkSpace.Instance.Solution.SolutionOperations.ConvertSolutionRelativePath(calculatedDestinationPath);
+            if(FileOperationMode == eFileoperations.UnZip && !calculatedDestinationPath.EndsWith("\\"))
```

Comment:
If condition is specific to UnZip, we have the switch case above for all operations, please try to use this condition there as this is generic function for all operations

Reply:
Changes Completed